lms/static/sass/base/_grid-settings.scss

```diff
@@ -7,7 +7,7 @@ $border-box-sizing: false;
 
 
 /* Breakpoints */
-$mobile: new-breakpoint(max-width 320px 4);
-$tablet: new-breakpoint(min-width 321px max-width 768px, 8);
+$mobile: new-breakpoint(max-width 540px 4);
```

Comment:
Why are we changing these? Seems like the right thing - perhaps we really want to ADD a breakpoint for $large-mobile to reflect this - 320px is still where the iPhone comes into play (@talbs - any opinion...?)